jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletContextHandler.java

```diff
@@ -293,6 +293,12 @@ public ServletContextApi newServletContextApi()
         return new ServletContextApi();
     }
 
+    @Override
+    public InvocationType getInvocationType()
+    {
+        return InvocationType.BLOCKING;
```

Comment:
This seems unrelated.

Reply:
No, it is related because without it Servlets were invoked with PC in the selector thread, which would fail a bunch of client transport tests that were using just 1 selector.